pkg/apis/acid.zalan.do/v1/crds.go

```diff
@@ -211,6 +211,9 @@ var PostgresCRDResourceValidation = apiextv1.CustomResourceValidation{
 								Type:   "string",
 								Format: "uuid",
 							},
+							"clone_target_timeline": {
+								Type: "string",
```

Comment:
But this is not the timeline *ID*

Reply:
@Sudeepta92 FxKu is right here. The actual ID can only be an integer. What you see in the WAL archive name is a hex value. So for instance `0000003400000000000000CD` would be timeline_id `52`. Nevertheless, one can also specify `latest` or `current` in the recovery.conf. Which is default if nothing is set.
So I think, you could use and `int32` here.

Reply:
Oh yeahh its a hex value in the end. I will change to int32.

Reply:
@FxKu @thedatabaseme btw, on changing type to integer back from string it fails when no timeline id is passed as it can no longer receive "latest"/"current".

Reply:
Ok, so have removed the possibility for string value "latest" and in such a scenario it will calcuate the latest timeline id as integer. You may ignore my previous comment.